crates/tabby/src/services/code.rs

```diff
@@ -137,10 +136,16 @@ fn merge_code_responses_by_rank(
 
     for (rank, bm25, doc) in compute_rank_score(bm25_resp).into_iter() {
         let chunk_id = get_chunk_id(&doc);
-        // Only keep items with embedding score > 0.
         if let Some((score, _)) = scored_hits.get_mut(chunk_id) {
             score.rrf += rank;
             score.bm25 = bm25;
+        } else {
+            let scores = CodeSearchScores {
```

Comment:
Is it a bug fix?

Reply:
No, it's just that we want to extract more results to the frontend for debugging purposes.